form/form.go

```diff
@@ -521,8 +525,8 @@ func BuildField(fileDescriptorSet *descriptor.FileDescriptorSet, msg *descriptor
 			s += '</div>';
 		s += setLink(json, "` + typName + `", "` + fieldname + `", "` + strings.Replace(help, "\n", "\\n", -1) + `");
 		`
-		} else {
-			return `s += '<div class="children" type="` + typName + `">';
+		}
```

Comment:
There are still a few unnecessary removing of elses, that don't help with readability of the diff.  I am all for doing this in a future diff.  It is just that when I have 5 minutes to review it makes it really hard to read the diff.

Reply:
Can we still remove these syntactic changes to make it easier for me to read?
Sorry it has been crazy busy.

Reply:
Done